extensions/resource-deployment/src/localizedConstants.ts

```diff
@@ -3,14 +3,16 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+//import { EOL } from 'os';
 import * as nls from 'vscode-nls';
 import { OptionsSourceType } from './helpers/optionSources';
 import { FieldType, OptionsType } from './interfaces';
 
 const localize = nls.loadMessageBundle();
 
 export const account = localize('azure.account', "Azure Account");
-export const subscription = localize('azure.account.subscription', "Subscription");
+export const subscription = localize('azure.account.subscription', "Subscription (selected subset)");
+export const subscriptionDescription = localize('azure.account.subscriptionDescription', "Only subscriptions selected in Azure navigation tree are shown. Change the set of selected subscriptions in 'CONNECTIONS->AZURE' navigation tree to see others)");
```

Comment:
Don't use capitals like this. My suggestion would be `Change the currently selected subscriptions through the 'Select Subscriptions' action on an account listed in the 'Azure' tree view of the 'Connections' viewlet.

Although I'm wondering if maybe we should also have a checkbox for displaying all subs or just filtered ones. Not sure on that though so let's go with this for now and if we get negative feedback we can look at it again then.

Reply:
later we can actually sync filtering done in deployment back to the azure viewlet as well. That is the experience that the azure portal has.